ANT+_Android_SDK/Sample/ANT+ Plugin Sampler/source/src/main/java/com/dsi/ant/antplus/pluginsampler/pulsezones/PulseZoneUtils.java

```diff
@@ -0,0 +1,63 @@
+package com.dsi.ant.antplus.pluginsampler.pulsezones;
+
+import android.os.Bundle;
+import android.content.Intent;
+
+/**
+ * Created by ksenia on 09.01.19.
+ */
+
+public class PulseZoneUtils {
```

Comment:
Util classes shouldn't have any state ... by state I mean class members. 
They should contain mainly static methods and don't have constructors .